presto-orc/src/test/java/com/facebook/presto/orc/TestCachingOrcDataSource.java

```diff
@@ -146,13 +153,16 @@ public void testTinyStripesReadCacheAt()
         DataSize maxMergeDistance = new DataSize(1, Unit.MEGABYTE);
         DataSize tinyStripeThreshold = new DataSize(8, Unit.MEGABYTE);
 
+        OrcAggregatedMemoryContext systemMemoryContext = new TestingHiveOrcAggregatedMemoryContext();
+
         TestingOrcDataSource testingOrcDataSource = new TestingOrcDataSource(FakeOrcDataSource.INSTANCE);
         CachingOrcDataSource cachingOrcDataSource = new CachingOrcDataSource(
                 testingOrcDataSource,
                 createTinyStripesRangeFinder(
                         ImmutableList.of(new StripeInformation(123, 3, 10, 10, 10, ImmutableList.of()), new StripeInformation(123, 33, 10, 10, 10, ImmutableList.of()), new StripeInformation(123, 63, 1048576 * 8 - 20, 10, 10, ImmutableList.of())),
                         maxMergeDistance,
-                        tinyStripeThreshold));
+                        tinyStripeThreshold),
+                systemMemoryContext.newOrcLocalMemoryContext(CachingOrcDataSource.class.getSimpleName()));
         cachingOrcDataSource.readCacheAt(3);
         assertEquals(testingOrcDataSource.getLastReadRanges(), ImmutableList.of(new DiskRange(3, 60)));
         cachingOrcDataSource.readCacheAt(63);
```

Comment:
Can we assert `assertEquals(systemMemoryContext.getBytes(), 8 * 1048576);` after this?